Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/SortAdapter.cpp

```diff
@@ -132,10 +230,19 @@ namespace AZ::DocumentPropertyEditor
 
             // sort the source child before copying it
             auto sortedChild = GetSortedValue(sourceValue[nextInSort], *nextSortedIter);
+            
+            // do the actual copy into the current position
             sortedValue[nextIndex] = sortedChild;
         }
 
         return sortedValue;
     }
 
+    void RowSortAdapter::ResortRow(Dom::Path sortedPath)
+    {
+        // TODO once the DPE supports move operations:
```

Comment:
suggestion: Rather than adding a TODO it might be better to refer to a GHI (or Jira Id) to know how this is tracked.